com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/health/HealthRetrieverTest.java

```diff
@@ -101,15 +106,49 @@ public void testGetHealth() {
 
         HealthCheckOperation healthCheckOperation = health.getHealthCheckOperations()
                                                           .get(0);
+        validateHealthCheckOperationParameters(healthCheckOperation);
+        Mockito.verify(operationQuery, times(1))
+               .user(USER_NAME);
+        validateMandatoryParametersAreSet();
+    }
+
+    @Test
+    void testGetHealthWithoutUsername() {
+        prepareConfiguration(ImmutableHealthCheckConfiguration.builder()
+                                                              .mtaId(MTA_ID)
+                                                              .spaceId(SPACE_ID)
+                                                              .timeRangeInSeconds(TIME_RANGE_IN_SECONDS)
+                                                              .build());
+        Health health = healthRetriever.getHealth();
+
+        assertTrue(health.isHealthy());
+        HealthCheckOperation healthCheckOperation = health.getHealthCheckOperations()
+                                                          .get(0);
+        validateHealthCheckOperationParameters(healthCheckOperation);
+        Mockito.verify(operationQuery, times(0))
```

Comment:
I think that there is `never()` or something similar, which can be used instead of `times(0)`.